features/step_definitions/schools/onboarding_steps.rb

```diff
@@ -5,7 +5,7 @@
 Given "I have completed the DBS Requirements step" do
   steps %(
     Given I am on the 'DBS requirements' page
-    And I choose 'Yes - Outline your DBS requirements' from the 'Do you require candidates to have or get a DBS check?' radio buttons
+    And I choose 'Yes - Outline your DBS requirements' from the 'Do candidates need to have or get a DBS check?' radio buttons
```

Comment:
This whole question needs re-designing, as per the content recommendations, so I think we should do the changes all in one go with dev help.